ChaosMod/Effects/db/Player/PlayerLagCamera.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+	Effect by Gorakh
+*/
+
+#include <stdafx.h>
+
+static int constexpr CAM_DELAY_NORMAL = 250;
+static int constexpr CAM_DELAY_VEHICLE = 1000;
+
+static int GetTargetCamDelay()
+{
+	return IS_PED_IN_ANY_VEHICLE(PLAYER_PED_ID(), true) ? CAM_DELAY_VEHICLE : CAM_DELAY_NORMAL;
+}
+
+struct CameraSnapshot
+{
+	int timestamp;
+
+	Vector3 position, rotation;
+	float fov;
+
+	CameraSnapshot()
+	{
+		timestamp = GET_GAME_TIMER();
+
+		position = GET_GAMEPLAY_CAM_COORD();
+		rotation = GET_GAMEPLAY_CAM_ROT(2);
+		fov = GET_GAMEPLAY_CAM_FOV();
+	}
+};
+
+std::vector<CameraSnapshot> cameraSnapshots;
```

Comment:
Make sure to add a `static` specifier here

Reply:
Fixed :)